src/state/reducers/members.js

```diff
@@ -14,6 +14,7 @@ const initialState = {
 
 export const loadMembers = createAction('LOAD_MEMBERS', (uid) => ({
   promise: new Promise((resolve, reject) => {
+    // TODO: check it actually reloads when the collection changes
```

Comment:
mh... feels the page doesn't reflect realtime updates. need to investigate more on this.